packages/gatsby-theme-docs/globals.css

```diff
@@ -1,6 +1,7 @@
 @import '@commercetools-uikit/design-system/materials/custom-properties.css';
+@import url('./src/fonts/Inter/inter.css');
```

Comment:
Why did you choose to move this into globals.css but leave the roboto mono import in gatsby-browser?   I can understand that globals could actually be the better place but why not move roboto mono here, too?   -> consistent would be better in any case. 
Also add a comment explaining why the fonts are handled differently to prevent a later ambitious maintainer from "cleaning up" and making it fontsource based again.  Thanks !